README.md

```diff
@@ -8,12 +8,22 @@ Interested in development? Join the discussion on the Eterna Discord!
 
 ## Components
 
-- [`@eternagame/bootstrap](./packages/bootstrap) - Bootstrap a new project using the Eternagame standards
-- [`@eternagame/nx`](./packages/nx) - Opinionated Nx utilities
-- [`@eternagame/tsconfig`](./packages/tsconfig) - Opinionated TypeScript configurations
-- [`@eternagame/eslint-plugin`](./packages/eslint-plugin) - Semi-opinionated ESLint configuration
-- [`@eternagame/jest`](./packages/jest) - Opinionated Jest utilities
-- [`@eternagame/vite`](./packages/vite) - Opinionated Vite configurations
+### Bootstrapping
+
+Want to set up a new project/repository using Eterna's standard structure? Check out [`@eternagame/bootstrap`](./packages/bootstrap)
+
+This will also set you up to use our nx plugin, [`@eternagame/nx`](./packages/nx), which has our standard nx configuration
+and package generators.
+
+### Configuration
+
+- [`@eternagame/tsconfig`](./packages/tsconfig) - Our standard `tsconfig`s
+- [`@eternagame/eslint-plugin`](./packages/eslint-plugin) - Our standard ESLint configuration
+- [`@eternagame/jest`](./packages/jest) - Our standard Jest configuration
+- [`@eternagame/vite`](./packages/vite) - Our standard Vite configurations
```

Comment:
Just a clarification: some of these are plural, and some are singular. Intentional? Is it that we have one Jest configuration, but potentially multiple Vite or tsconfig configurations?

Reply:
Well there are multiple tsconfigs, and vite can be configured multiple ways - though I guess the jest config can be configured multiple ways too! I guess the question here would be "does a parameterized config count as one config, or many?" Any thoughts?

Reply:
When I read standard tsconfigs, I assume there are multiple tsconfig files. So a single config file that has parameters that can be adjusted feels like one config. Whereas two config files for different environments feels like multiple configs. That's a weird delineation, but it's what I naturally think here.